client/webserver/site/src/js/app.js

```diff
@@ -498,8 +498,8 @@ export default class Application {
       case 'walletstate':
       case 'walletconfig': {
         const wallet = note.wallet
-        this.assets[wallet.assetID].wallet = wallet
-        this.walletMap[wallet.assetID] = wallet
+        if (this.assets !== undefined) this.assets[wallet.assetID].wallet = wallet
+        if (this.walletMap !== undefined) this.walletMap[wallet.assetID] = wallet
```

Comment:
Not sure about this. What are you fixing?

Reply:
You are right. Removed it